src/users/users.php

```diff
@@ -368,8 +368,22 @@ public function get()
 		else
 		{
 			$model = new UsersModelUsers;
+
+			$app = JFactory::getApplication('administrator');
+
+			$app->setUserState("com_users.users.default.filter.search", $input->get('search'));
+			$app->setUserState("com_users.users.default.filter.active", '0');
+			$app->setUserState("com_users.users.default.filter.state", '0');
+
 			$users = $model->getItems();
 
+			if (!$users)
+			{
```

Comment:
@Pratikshakhandagale @thite-amol This is a valid request, right?
Shall this be code 200, with message as `No matching users` ?

Reply:
I meant return regular response with no users, and a message?